packages/react-core/src/components/Card/examples/Card.md

```diff
@@ -109,37 +109,39 @@ A common use case of this is to set all but one body section to `isFilled={false
 
 ### Selectable
 
-A card can be selected by clicking anywhere within the card.
+A selectable card can be selected by clicking anywhere within the card.
 
-You must avoid rendering any other interactive content within the `<Card>` when it is meant to be selectable only. Refer to our [clickable and selectable example](#clickable-and-selectable-cards) if you need a card that is both selectable and has other interactive content.
+You must avoid rendering any other interactive content within the `<Card>` when it is meant to be selectable only. Refer to our [actionable and selectable example](#clickable-and-selectable-cards) if you need a card that is both selectable and has other interactive, actionable content.
 
 ```ts file='./CardSelectable.tsx'
 
 ```
 
 ### Single selectable
 
-When a group of single selectable cards are related, you should pass the same `name` property to each card's `selectableActions` property.
+When a group of single selectable cards are related, you must pass the same `name` property to each card's `selectableActions` property.
 
 ```ts file='./CardSingleSelectable.tsx'
 
 ```
 
-### Clickable
+### Actionable
 
-A card can perform an action or navigate to a link by clicking anywhere within the card. You can also pass in the `isExternalLink` property to `selectableActions` if you want a clickable card's link to open in a new tab or window.
+An actionable card can perform an action or navigate to a link by clicking anywhere within the card. You can also pass in the `isExternalLink` property to `selectableActions` if you want an actionable card's link to open in a new tab or window.
```

Comment:
@edonehoo do you mind taking a look at this copy.

Reply:
I like it! I might further reword it to be a little more user action oriented like:

"An actionable card can perform an action or navigate to a link when a card is clicked. To open a link in a new tab or window, pass the `isExternalLink` property to `selectableActions`. "

Reply:
Update the verbiage, though I kept in the "by clicking anywhere within the card" bit since the I mention it for the selectable card, and it's one of the differentiating aspects of the "selectable" only/"actionable" only cards and the "selectable and actionable" cards. Can also just remove that verbiage from both example descriptions intead, though.

Reply:
ah, I see what you mean. It also makes to me to leave your original phrasing in that case!